src/Files.App/Views/Shells/BaseShellPage.cs

```diff
@@ -567,18 +567,29 @@ public virtual void Back_Click()
 			var previousPageContent = ItemDisplay.BackStack[ItemDisplay.BackStack.Count - 1];
 			HandleBackForwardRequest(previousPageContent);
 
-			if (previousPageContent.SourcePageType == typeof(HomePage))
-				ItemDisplay.GoBack(new EntranceNavigationTransitionInfo());
-			else
-				ItemDisplay.GoBack();
+			try
+			{
+				ItemDisplay.GoBack(new SuppressNavigationTransitionInfo());
+			}
+			catch (COMException ex)
+			{
+				App.Logger.LogWarning(ex, ex.Message);
+			}
```

Comment:
Is it intentional to eliminate the if-else?

Reply:
Yes, we don't need the `EntranceNavigationTransitionInfo`